src/compas_view2/scene/camera.py

```diff
@@ -219,22 +229,7 @@ def viewworld(self):
         The view-world matrix transforms the scene from world coordinates to camera coordinates.
 
         """
-        T2 = Translation.from_vector([self.tx, self.ty, -self.distance])
-        T1 = Translation.from_vector(self.target)
-        if self.view.current == self.view.PERSPECTIVE:
-            Rx = Rotation.from_axis_and_angle([1, 0, 0], radians(self.rx))
-            Rz = Rotation.from_axis_and_angle([0, 0, 1], radians(self.rz))
-            R = Rx * Rz
-        elif self.view.current == self.view.FRONT:
-            R = Rotation.from_axis_and_angle([1, 0, 0], radians(-90))
-        elif self.view.current == self.view.RIGHT:
-            Rx = Rotation.from_axis_and_angle([1, 0, 0], radians(-90))
-            Rz = Rotation.from_axis_and_angle([0, 0, 1], radians(+90))
-            R = Rx * Rz
-        elif self.view.current == self.view.TOP:
-            R = Rotation()
-        else:
-            raise NotImplementedError
-        T0 = Translation.from_vector([-self.target[0], -self.target[1], -self.target[2]])
-        W = T2 * T1 * R * T0
-        return np.asfortranarray(W, dtype=np.float32)
+        T = Translation.from_vector(self.position)
+        R = Rotation.from_euler_angles(self.rotation)
+        W = T * R
+        return np.asfortranarray(W.inverted(), dtype=np.float32)
```

Comment:
`np.float32`?

Reply:
np.float32 was inherited from the previous code you wrote in the beginning, I was assuming it has something to do with gpu memory optimization?

Reply:
ha yes indeed